tests/adapters/switches/dell10g_test.py

```diff
@@ -293,6 +293,26 @@ def test_get_vlan_with_unknown_number(self):
 
         assert_that(str(expect.exception), equal_to("Vlan None not found"))
 
+    def test_get_vlan_interfaces(self):
+        self.mocked_ssh_client.should_receive("do").with_args("show vlan id 1000", wait_for=("--More-- or (q)uit", "#"), include_last_line=True).once().ordered().and_return([
+            "VLAN       Name                         Ports          Type      Authorization",
+            "-----  ---------------                  -------------  -----     -------------",
+            "1000   MyVlanName                       ch2-3,ch5-6,   Static    Required",
```

Comment:
My I suggest that we test with "ch2-3,ch5-7" and assert that we get 'port-channel 2', 'port-channel 3', 'port-channel 5', 'port-channel 6', 'port-channel 7' ?

Just to make sure...